src/components/styles/PageSection.styled.ts

```diff
@@ -12,23 +12,42 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right' }>`
-  display: grid;
+export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right'; responsiveReverse?: boolean }>`
+  display: flex;
   justify-content: space-between;
-  grid-template-rows: minmax(0, 1fr);
-  grid-template-columns: ${(props) => props.imgPosition && 'auto'} auto;
-  grid-template-areas: '${(props) => props.imgPosition === 'left' && 'image'} text ${(props) =>
-    props.imgPosition === 'right' && 'image'}';
-  gap: 5rem;
   animation-duration: 1s;
   animation-name: ${fadeInUp};
+
+  @media screen and (min-width: 700px) {
```

Comment:
this might not be required? i think can leave the grid implementation at the top then max-width media query to include display: flex instead?

Reply:
Ah I thought the grid-template-rows/columns/areas will affect the display but it didn't so I will move it back to main implementation. I leave the gap untouched as it will affected the gap in the display.